azure-pipelines/simple_test_framework.yml

```diff
@@ -44,30 +44,37 @@ jobs:
         displayName: "Install TARDIS environment"
 
       - bash: |
-              echo git clone $(ref.data.github.url) $(ref.data.home)
-              git clone $(ref.data.github.url) $(ref.data.home)
-              cd $(ref.data.home)
-              git fetch origin
-              git checkout origin/master
-              git lfs pull --include="atom_data/kurucz_cd23_chianti_H_He.h5" origin
-              git lfs pull --include="atom_data/chianti_He.h5" origin
-              git lfs pull --include="plasma_reference/" origin
-              git lfs pull --include="unit_test_data.h5" origin
-              git lfs pull --include="packet_unittest.h5" origin
-              echo MD5: `md5sum unit_test_data.h5`
+          source activate tardis
+          python setup.py build_ext --inplace
+        displayName: "TARDIS build"
+
+      - bash: |
+          GIT_LFS_SKIP_SMUDGE=1 git clone $(ref.data.github.url) $(ref.data.home)
+          cd $(ref.data.home)
+          git fetch origin
+          git checkout origin/master
+          git lfs pull --include="atom_data/kurucz_cd23_chianti_H_He.h5" origin
+          git lfs pull --include="atom_data/chianti_He.h5" origin
+          git lfs pull --include="unit_test_data.h5" origin
+          git lfs pull --include="packet_unittest.h5" origin
+          echo MD5: `md5sum unit_test_data.h5`
         displayName: "Fetch reference data"
+        condition: false
 
```

Comment:
I just removed the `plasma_reference` folder (does not exist in tardis-sn/tardis-refdata repo) and disabled the step with the `false` condition. The rest of the lines were moved by the YAML autoformatter.